test/test_shared_ng.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TEST_SHARED_NG_H
+#define TEST_SHARED_NG_H
+
+/* Test definitions that can be used in the zlib-ng build environment. */
```

Comment:
I'm not sure, but "can only be used" would be more accurate due to including `test_shared.h` right below... Also would be more accurate if placed after that include statement.

Reply:
Done.